items/src/main/scala/weco/api/items/services/SierraItemUpdater.scala

```diff
@@ -22,7 +24,11 @@ import scala.concurrent.{ExecutionContext, Future}
   *  can be placed on an item.
   *
   */
-class SierraItemUpdater(sierraSource: SierraSource)(
+class SierraItemUpdater(
+  sierraSource: SierraSource,
+  venueOpeningTimesLookup: VenueOpeningTimesLookup,
+  clock: LondonClock
```

Comment:
I'd be tempted to make use of a [java `Clock`](https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html) here, which in tests can be provided [as a fixed time](https://mincong.io/2020/05/24/java-clock/).